build/index.js

```diff
@@ -319,26 +319,21 @@ async function buildDocument(document, documentOptions = {}) {
         `MacroInvocationError trying to parse ${error.filepath}, line ${error.line} column ${error.column} (${error.error.message})`
       );
     }
-
     // Any other unexpected error re-thrown.
     throw error;
   }
 
-  const sampleIds = kumascript.getLiveSampleIDs(
-    document.metadata.slug,
+  let $ = cheerio.load(`<div id="_body">${renderedHtml}</div>`);
```

Comment:
I should have made this a `const` when I fixed the merge conflicts. Doing that now. 
```suggestion
  const $ = cheerio.load(`<div id="_body">${renderedHtml}</div>`);
```